server/src/main/java/com/cloud/resource/ResourceManagerImpl.java

```diff
@@ -203,6 +156,52 @@
 import com.cloud.vm.dao.UserVmDetailsDao;
 import com.cloud.vm.dao.VMInstanceDao;
 import com.google.gson.Gson;
+import org.apache.cloudstack.annotation.AnnotationService;
```

Comment:
can you undo the import re-order please?

Reply:
Done!